execution/executor/src/components/chunk_output.rs

```diff
@@ -116,9 +132,13 @@ impl ChunkOutput {
     #[cfg(not(feature = "consensus-only-perf-test"))]
     fn execute_block<V: VMExecutor>(
         transactions: Vec<Transaction>,
-        state_view: &CachedStateView,
+        state_view: Arc<CachedStateView>,
```

Comment:
Why do we need Arc?

Reply:
So, this is because the ExecutorShard runs as a separate thread and there is no way to pass a reference to that thread without lifetime issues.

Reply:
Yes, we were seeing around 30% performance degradation with that implementation. I still need to dig into the root cause of that but I think this is clean as well?